gitlab-base.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: gitlab-base
+  # The release-monitor identified is for the gitlab-runner,
+  # but these seem to be co-versioned.
```

Comment:
Would it help to make this a subpackage of gitlab-runner then? That way the wolfibot updates always keep these in sync, instead of needing two PRs.

Reply:
Generally this is the part where I feel a bit more uncertain of what we should do. Either create a project for this and make subpackages of it, as there are some images using it as base image.

Happy to hear your feedback as a noob.

Reply:
@mattmoor have triaged these images. Perhaps he knows what we should better do for this one.

Reply:
posting this here for reference: https://github.com/wolfi-dev/os/blob/main/gitlab-cng.yaml

looks like this sets us up to reference the scripts as runtime deps, for example: `gitlab-cng-base-scripts` essentially replicates what this PR is doing

Reply:
Yeah, I don't think base needs any packages we don't have.  It should just be assembling the image from assorted packages like the `gitlab-cng-base-scripts`, `xtail`, and `gitlab-logger` packages (this is off the top of my head, I'd consult the Dockerfile).

Reply:
@imjasonh they are in different repos :-/

Reply:
Yes, I talked with Josh and I'll create the base image based on the CNG image with the required deps.